### PR TITLE
[ADD] no-write-in-compute: Don't write on a computed field 

### PR DESCRIPTION
**Pull request by moylop260**
Fix https://github.com/OCA/pylint-odoo/issues/356

### CODE REVIEW

File: src/pylint_odoo/checkers/odoo_addons.py [hunk 1]
```diff
@@ -170,6 +170,7 @@
         CHECK_DESCRIPTION,
     ),
     "E8130": ("Test folder imported in module %s", "test-folder-imported", CHECK_DESCRIPTION),
+    "E8135": ("Compute method calling 'write' method", "compute-write", CHECK_DESCRIPTION),
```
Comment (moylop260) [inline src/pylint_odoo/checkers/odoo_addons.py:173]: @pedrobaeza @luisg123v 

Could you help me to define a better and more intuitive message and check-name here, please?

Comment (pedrobaeza) [reply src/pylint_odoo/checkers/odoo_addons.py:173]: I think this can serve:

`"Compute method calling 'write'. Use 'update' instead."`

Comment (moylop260) [reply src/pylint_odoo/checkers/odoo_addons.py:173]: Good!

Thank you!

And name-of-check?

Comment (pedrobaeza) [reply src/pylint_odoo/checkers/odoo_addons.py:173]: `no-write-in-compute`?

**Comment by moylop260** [comment]
@voronind  

I have added all the test-cases that you mentioned in the issue

Please, could you check it

Disclaimer: The code is still work-in-progress but the POC is enough to demo it is possible

**Pull request merged**